core/base/src/constants/rpc.ts

```diff
@@ -91,6 +91,7 @@ const rpcConfig = [[
     ["Snaxchain",       "https://testnet.snaxchain.io"],
     ["Unichain",        "https://sepolia.unichain.org"],
     ["Worldchain",      "https://worldchain-sepolia.g.alchemy.com/public"],
+    ["MonadDevnet",     "TBD"],
```

Comment:
oof. what would @kev1n-peters recommend here?

Reply:
Maybe we could set it to the empty string and add a comment saying, "TODO: no public rpc is currently available, override with a custom rpc for now"